### PR TITLE
Add security headers

### PR DESCRIPTION
**Pull request by mraible**
The score for the blog is currently a D. Hopefully this helps fix that.

Screen Shot 2021-09-07 at 12 39 10 PM

### CODE REVIEW

**Review by bdemers — APPROVED** [review]
does this work with the AWS S3 deployment too?

**Comment by mraible** [comment]
@bdemers It's funny you should ask. The whole reason I started this PR was because I talked with @hamzaessaoui-okta. He mentioned:

> I worked with Randall to move it to Netlify and I believe he kept AWS as a backup.

This isn't quite working yet. I've been testing with @jpf's [Password Hashing post](https://developer.okta.com/blog/2021/03/05/ultimate-guide-to-password-hashing-in-okta). Once it loads w/o errors, I'll merge and we'll see if it works in production.

**Pull request merged**
